app.py

```diff
@@ -35,6 +35,7 @@ def result():
     response = request.get_json()
     response["page"] = 1
     response["page_ads_number"] = 10
+    print(response)
```

Comment:
Do you need print here?

Reply:
Not really, but in this repo we don't have logger, and I want to see input params in terminal)

Reply:
Oh, ok